tailwind.config.js

```diff
@@ -27,7 +30,8 @@ module.exports = {
         11.5: '4.5rem',
         12: '5.938rem',
         13: '6.061rem',
-        14: '11.644rem',
+        14: '6.5rem',
```

Comment:
Are there any lines affected by the fontSize 14 change

Reply:
Nope, it effected only `RetroText` on that I was working